assets/src/scripts/components/hydrograph/audible.spec.js

```diff
@@ -81,11 +78,13 @@ describe('audibleUI', () => {
         expect(store.getState().timeSeriesState.audiblePlayId).not.toBeNull();
     });
 
-    it('Expects the store to have a null playId if the  button is clicked after clicking the once', () => {
+    it('Expects the store to have a null playId if the  button is clicked after clicking the once', (done) => {
         container.select('button').dispatch('click');
-        container.select('button').dispatch('click');
-
-        expect(store.getState().timeSeriesState.audiblePlayId).toBeNull();
+        window.requestAnimationFrame(() => {
```

Comment:
Anytime you check the state of the DOM after an update to Redux, you now need to wait until the next animation frame due to the throttling. You will see this pattern repeat in the tests.